app/models/revision.rb

```diff
@@ -1,12 +1,13 @@
 # frozen_string_literal: true
 
-# A model that represents a particular revision of a document. An edition
-# always has a revision for the current state of it and then there are past
-# revisions that represent all the changes a document has been through.
+# Represents a particular revision of a document - acting as a snapshot to
+# a particular users edit.
 #
 # This model aims to store as little data as possible (since there are many
```

Comment:
Not sure the argument is clear from saying 'there are many in the database'. Perhaps: 'This model stores as little data as possible by having associations to more specific types of revision and delegating its methods to them.'

Reply:
Sounds good.